content/en/docs/Tasks/creating-a-catalog.md

```diff
@@ -9,6 +9,7 @@ description: >
 
 - [opm](https://github.com/operator-framework/operator-registry/releases) `v1.19.0+`
 
+>Note: This document discusses creating a catalog of operators using plaintext files to store catalog metadata, which is the [latest feature][file-based-catalog-spec] of OLM catalogs. If you are looking to build catalogs using a sqllite database to store catalog metadata instead, please read the [v0.18.z version][v0.18.z-version] of this doc instead. 
```

Comment:
```suggestion
>Note: This document discusses creating a catalog of operators using plaintext files to store catalog metadata, which is the [latest feature][file-based-catalog-spec] of OLM catalogs. If you are looking to build catalogs using the deprecated sqlite database format to store catalog metadata instead, please read the [v0.18.z version][v0.18.z-version] of this doc instead. 
```

Reply:
I do not think that a person outside of the OLM team will actually know what that means.  How an Operator author will actually know what means SQL index and etc? The audience of this doc is Operator Authors who are looking for how to work with upgrade index and publish their solutions on available catalogs. In this way, (ihmo) we need to make clear what is SQL index and file config index and that the world probably is not prepared yet to adopt the new format. 

My suggestion here would be something like : 

## Announcement

Note that after the OLM release `0.18.x` version the format that has been used to build the index (`SQL format`) was deprecated and the [file-based catalogs][file-based-catalog-spec] was introduced.

It is expected that you will only use the new format which brings more flexibility in the future, however, that might takes time for some OLM adopters began to migrate their solutions and let you start to take advantage of it.

In this way, if you are looking to publish your solution to vendor-specific catalogs such as OpenShift, OKD or are only  looking for how you could create the upgrade graph for the previous format please see the previous version of this document for the [v0-18-z](https://v0-18-z.olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/) release. 

c/c @bparees @anik120 @joelanford